notebooks-accounting/templates/ssm-config.yaml

```diff
@@ -50,6 +50,9 @@ data:
     # path_type: dirq
     # Outgoing messages will be read and removed from this directory.
     path: {{ .Values.ssm.messaging.path | default "/accounting/ssm" }}
+{{- if .Values.ssm.messaging.token }}
```

Comment:
does this substitute the certificate above?

Reply:
It looks like both are needed token and certificate (the ssm sender still requires the certificate files options and the certificate files must be there).
I guess the "connection" with the GOCDB configuration is still used - DN is configured there for accounting.